### ISSUE TITLE
bug in rotm2quat.m

### ISSUE THREAD

**Issue by ronb127** [robotology/mex-wholebodymodel#92]
Line 35:
quat(1,1) = (rotm(3,2) + rotm(2,3))*s_inv;
Should be:
quat(1,1) = (rotm(3,2) - rotm(2,3))*s_inv;

**Comment by traversaro** [comment]
Thanks for reaching out @ronb127 ! 
cc @Ganimed

**Comment by Ganimed** [comment]
Thank you, I will check this!

**Comment by Ganimed** [comment]
I have checked the formulas. Mainly I have used the formulas from the book of [Jaz10] and in equation (3.150) at page 110 there is the same sign-bug. 

Here for example is the correct calculation of this part: http://www.euclideanspace.com/maths/geometry/rotations/conversions/matrixToQuaternion/.

Thank you for the hint I have corrected this in my branch.

[Jaz10]: Jazar, Reza N.: Theory of Applied Robotics: Kinematics, Dynamics, and Control. 2nd Edition, Springer, 2010, p. 110, eq. (3.149)-(3.152). 

@traversaro: I need the owner-rights that I can merge it to the master, thank's.
